jobs/build-webapp.groovy

```diff
@@ -528,6 +528,18 @@ def uploadGraphqlSafelist() {
 }
 
 
+def createCloudRunTags(){
+   echo("Creating Cloud Run tags.")
+   dir("webapp") {
+      try {
+         exec(["deploy/update_cloud_run_tags.py", NEW_VERSION]);
```

Comment:
You need to pass in the list of services being deployed as the 'ignore-services' arg.